LICENSE.txt

```diff
@@ -1,4 +1,4 @@
-Copyright (c) 2009-2010 Sebastian Kraemer, basti.kr@gmail.com
+Copyright (c) 2009-2017 Sebastian Kraemer, basti.kr@gmail.com
```

Comment:
This is fine to change the Copyrights, but this is typically NOT needed. This should be done when the code is updated and touched not as a blanket thing IMHO.